puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_17_stable.yaml

```diff
@@ -0,0 +1,45 @@
+- job:
+    name: test_proxy_1_17_stable
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 16
+    properties:
+      - github:
+          url: https://github.com/theforeman/smart-proxy
+    scm:
+      - foreman-proxy:
+          branch: '1.17-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.0
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
+      - axis:
+          type: user-defined
+          name: puppet
+          values:
+          - 3.8.0
+          - 4.10.9
+          - 5.3.3
+    execution-strategy:
+      combination-filter: '!( (ruby ==~ /2\.[^0]*/ && puppet ==~ /3\.[0-4].*/) || (ruby ==~ /2\.[2-9].*/ && puppet ==~ /3.*/) || (ruby ==~ /2\.[3-9].*/ && puppet ==~ /4\.[0-3].*/) )'
```

Comment:
This needs an update now. I'm not sure what we exactly want, but I think:
* 3.8.0 + 2.0-2.1
* 4.10.9 + 2.1-2.4
5.3.3 + 2.4